### ISSUE TITLE
Imposible to get float params from url string

### ISSUE THREAD

**Issue by ChronicusUA** [visionmedia/express-resource#60]
Example map:

thingsResource.map('get', '/:min/:max', thingsRouts.getMinMax) 

Min and max is float. 
I'm making 'get' request to url: things/12.22/12.45 or things/12.22/12.45/
I expected: req.params.min = 12.22 and req.params.max = 12.45, but i get: req.params.max = 12 and req.params.format = 45 instead

Easy workarouund is just make request like: things/12.22/12.45. <- this point is required. 

Sorry for my bad english :)

**Comment by tj** [comment]
yeah that's the downside of supporting "formats" in the pathname (".json" etc), where Accept is really what should be used. IMO we should remove support for them or whitelist specific ones

**Comment by ChronicusUA** [comment]
Yes, I understand it. But may be for float values you could do follow: get format and do parseInt for format if is number - you could thing that is float ;)
And for text values with point, may be use something like '\.'
